xbmc/pvr/addons/PVRClient.cpp

```diff
@@ -9,6 +9,7 @@
 #include "PVRClient.h"
 
 #include "ServiceBroker.h"
+#include "addons/kodi-addon-dev-kit/include/kodi/addon-instance/PVR.h" // added for compile test on related sources only!
```

Comment:
@ksooo have removed everywhere the include of complete PVR.h.

Here is the only place now where added and only to have a place where it's compile tested.

Think there need a todo for special cmake project on kodi's project for addons, to check them and not via this :smile:

Reply:
Nice. Thanks you so much.

Reply:
NP, also planned to add (after the "C" is safe on all places) a check system about, to confirm it stays as "C", then I add also a second system to make a small compile test with C++ headers.

So that for both places compile is confirmed.